src/app.js

```diff
@@ -1 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { terminal } = require('./terminal');
+const { generateNumber } = require('./numberGenerator');
+const { checkAnswer } = require('./checkAnswer');
+
+function gameStart() {
+  terminal.question('Enter a number: ', (userInput) => {
+    if (userInput.length !== 4 || !Number(userInput)) {
+      console.log('Invalid number, please enter a 4 different digits');
+      terminal.question('Enter a number: ', gameStart);
+
+      return;
+    }
+
+    const searchedNumber = generateNumber();
```

Comment:
if user did not win, this number will be generated again and overwrite? In this case it is possible to win only by guessing all the numbers at once.